.github/workflows/ci.yml

```diff
@@ -88,7 +88,7 @@ jobs:
     - name: Install python dependencies
       run: |
         sudo apt-get install graphviz graphviz-dev
-        pip install wheel numpy scipy f90nml h5py scikit-build cmake qsc sympy pyevtk matplotlib ninja plotly networkx pygraphviz
+        pip install wheel numpy scipy f90nml h5py scikit-build cmake qsc sympy pyevtk matplotlib ninja plotly networkx pygraphviz mpi4py  py_spec pyoculus h5py
```

Comment:
h5py is listed twice here. 

I'm trying to remember why all these packages are manually `pip install`ed here. Wouldn't they automatically be installed when simsopt is `pip install`ed?

Reply:
I think it was the holdover from the initial days, when we were building wheels first and then installing from the wheels. Still that does not explain the reason. Let me clean it up.

Reply:
I checked it again. We are installing them because SPEC build requires them. SPEC installation setup is stagnant. I implemented a minimal setup when building the Python wrappers. No one updated it afterward.